postgraphile/package.json

```diff
@@ -7,6 +7,7 @@
     "lint": "tslint --project ./tsconfig.json --config ./tslint.json",
     "postinstall": "rm -rf node_modules/@graphile && mkdir node_modules/@graphile && cp -R ./vendor/postgraphile-supporter/ ./node_modules/@graphile/plugin-supporter/",
     "start": "npm run build && node ./build/dist/vulcanize-postgraphile-server.js",
+    "dev": "./node_modules/typescript/bin/tsc && CONFIG_PATH='./config.toml' node build/dist/src/index.js",
```

Comment:
do we want to set the `CONFIG_PATH` env variable here? seems like it could override one that's defined by the user, which might be confusing. also it seems like the user may need to create this file since I think it'll be ignored by git?

Reply:
It doesn't need to be defined, here, was just for dev purposes - will remove. And yup, currently the user would need to create the config file/the config file is optional